File: CMS/src/app/shared/ProvinceData.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ProvinceData } from 'src/app/shared/ProvinceData.model';
import { BaseService } from './Base.service';
@Injectable({
  providedIn: 'root'
})
export class ProvinceDataService extends BaseService {

  List: ProvinceData[] | undefined;
  ListFilter: ProvinceData[] | undefined;
  FormData!: ProvinceData;

  constructor(public httpClient: HttpClient) {
    super(httpClient);
    this.Controller = "ProvinceData";
  }
}

File: Service/Implement/ProvinceDataService.cs
namespace Service.Implement
{
    public class ProvinceDataService : BaseService<ProvinceData, IProvinceDataRepository>
    , IProvinceDataService
    {
    private readonly IProvinceDataRepository _ProvinceDataRepository;
    public ProvinceDataService(IProvinceDataRepository ProvinceDataRepository) : base(ProvinceDataRepository)
    {
    _ProvinceDataRepository = ProvinceDataRepository;
    }
    }
    }

